RUFAS/output_manager.py

```diff
@@ -317,6 +318,29 @@ def _exclude_info_maps(self, pool: Dict[str, pool_element_type]) -> Dict[str, po
                 value.pop("info_maps")
         return pool_copy
 
+    def _load_txt_file_names_to_list(self, dir_path: str) -> List[str]:
+        """ Looks in directory for txt file names and adds them to a list.
+
+        Parameters
+        ----------
+        path : str
+            Path of the directory to be searched.
+
+        Returns
+        -------
+        List
+            A list of txt file names.
+
+        """
+        dir_path = Path(dir_path)
+        if not dir_path.is_dir():
+            raise IsADirectoryError("specified path is not a directory")
+        txt_files = []
+        for filename in os.listdir(dir_path):
```

Comment:
```suggestion
        all_files = os.listdir(dir_path)
        for filename in all_files:
```
I'm not sure if python is intelligent enough to do caching here. If it doesn't, having os.listdir() in the for loop can lead to multiple requests to the disk, which is not good. Better safe than sorry, let's make sure it doesn't happen.